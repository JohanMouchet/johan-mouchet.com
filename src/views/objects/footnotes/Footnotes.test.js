import React from "react";
import { shallow } from "enzyme";
import { FootnotesReference, FootnotesNotes } from ".";

it("Renders <FootnotesReference /> without crashing", () => {
  shallow(<FootnotesReference />);
});

it("Renders <FootnotesNotes /> without crashing", () => {
  shallow(<FootnotesNotes />);
});
