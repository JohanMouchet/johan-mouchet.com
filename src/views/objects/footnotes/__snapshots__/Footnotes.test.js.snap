// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <FootnotesNotes /> without crashing 1`] = `""`;

exports[`Renders <FootnotesReference /> without crashing 1`] = `
<sup>
  <a
    className="o-footnote-ref"
    href="#footnote:undefined"
    id="footnote-ref:undefined"
  />
</sup>
`;
